docs/detections/alerts-reduce.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[reduce-notifications-alerts]]
+== Reduce notifications and alerts
+
+{elastic-sec} offers several features to help reduce the amount of notifications and alerts produced by your detection rules. This table provides a general comparison of these features, with links for more details:
+
+[cols="2"]
+|===
+
+| <<snooze-rule-actions,Rule action snoozing>>
+a| *_Prevents a specific rule's notification actions from running_*. 
```

Comment:
I think "stop" is the more accurate verb because "prevent" seems more preemptive. I could be wrong, but I don't think users would proactively snooze action notifications after enabling a rule because they thought the rule would be noisy. 

```suggestion
a| *_Stops a specific rule's notification actions from running_*. 
```